Lib/axisregistry/__init__.py

```diff
@@ -168,14 +168,20 @@ def build_stat(ttFont, sibling_ttFonts=[]):
     fvar_nameids = set(i.subfamilyNameID for i in fvar.instances)
     if "STAT" in ttFont:
         stat = ttFont["STAT"]
-        axis_values = stat.table.AxisValueArray.AxisValue
-        axes = stat.table.DesignAxisRecord.Axis
-        for ax in axis_values:
-            if ax.ValueNameID not in fvar_nameids:
-                nametable.removeNames(nameID=ax.ValueNameID)
-        for ax in axes:
-            if ax.AxisNameID not in fvar_nameids:
-                nametable.removeNames(nameID=ax.AxisNameID)
+        try:
+            axis_values = stat.table.AxisValueArray.AxisValue
+            for ax in axis_values:
+                if ax.ValueNameID not in fvar_nameids:
+                    nametable.removeNames(nameID=ax.ValueNameID)
+        except:
```

Comment:
This is a bit of a big hammer. I would wrap the try/except block around the `removeNames` and check for a particular exception.

Reply:
Thanks.

I originally wrapped these in try/accepts because the attribs don't exist if the counts of either of these arrays is 0. I've opted to just check that both counts are greater than 0 which feels far cleaner.